### PR TITLE
fix(lib): typo in error message

### PR DESCRIPTION
**Pull request by Kerumen**
**The PR is now only a fix for a typo problem.**

---------------

It's useless to add an `await` when we already return a promise.

See the eslint rule for example: https://eslint.org/docs/rules/no-return-await

(also fixed a small typo)

Edit: initially I removed both 2 `await`s but as the first one is required to check the error, I created a variable to remove the `return await`.

Tell me what do you think :)

### CODE REVIEW

**Comment by vvo** [comment]
Hey there thanks for the PR, I was curious so I did my research too on this subject. It seems that the world of JavaScript developers is not quite sure about whether or not `return await` is an issue in term of performance and debugging information.

See:
- https://github.com/eslint/eslint/issues/12246
- https://github.com/standard/standard/issues/1442
- https://stackoverflow.com/questions/44806135/why-no-return-await-vs-const-x-await/44806230#44806230
- and even a previous discussion from the last place I worked at! (I forgot about it): https://github.com/algolia/eslint-config-algolia/issues/78

To be honest, unless you find a bug or performance issue caused by the lines you updated, I won't be merging this.

I still welcome your changes on the typos on the errors though :)

Also, pure stylistic but I like to know whether or not a method is an async one. If you remove await from await ironStore then you now need to know that ironStore is an async function.

Thanks anyway and I hope you'll submit more PRs here!

File: lib/index.js [hunk 1]
```diff
@@ -80,16 +80,17 @@ export default function withIronSession(
 
 async function getOrCreateStore({ sealed, password, ttl }) {
   try {
-    return await ironStore({
+    const store = await ironStore({
```
Comment (vvo) [inline lib/index.js:83]:
```suggestion
    return await ironStore({
```

File: lib/index.js [hunk 2]
```diff
@@ -80,16 +80,17 @@ export default function withIronSession(
...
       sealed,
       password,
       ttl,
     });
+    return store;
```
Comment (vvo) [inline lib/index.js:88]:
```suggestion
```

File: lib/index.js [hunk 3]
```diff
@@ -80,16 +80,17 @@ export default function withIronSession(
...
   } catch (err) {
     if (err.message === "Expired seal") {
       // if seal expires and it somehow reaches our servers (seal was stolen, bad clock)
       // then we just start a new session over
-      return await ironStore({ password, ttl });
+      return ironStore({ password, ttl });
```
Comment (vvo) [inline lib/index.js:93]:
```suggestion
      return await ironStore({ password, ttl });
```

**Comment by vvo** [comment]
The whole "no-return-await" rule/discussion seems to boil down to:
- "This is incorrect code!" (from no-return-await peeps)
- "This is less performant" (from no-return-await peeps)
- "This is less clear" (from both side)

To confirm, unless we can prove it's significantly slower in real-world conditions (not in a loop trying to produce a significant slowdown). This is a stylistic choice and one that for now I haven't jumped on so we'll keep this part as it is, thanks!

**Comment by Kerumen** [comment]
Thanks for this overview on this topic, I was "blindly" following my eslint rule.

Sorry for the useless PR, I'll accept your changes to commit only the typo fix.

**Pull request merged**